.github/workflows/e2e.yml

```diff
@@ -59,12 +60,24 @@ jobs:
         export KUBECONFIG=${HOME}/.kube/config
         make test-e2e
 
+    - name: Export E2E Tests logs
+      if: ${{ failure() }}
+      run: |
+        mkdir -p ${LOG_DIR}
+        # tail --lines=+2 - do not print first line
+        # awk  '{ print $1 }' - print only first columns
+        for pod in $(kubectl get pods -n $E2E_NAMESPACE_NAME | tail --lines=+2 | awk  '{ print $1 }') 
```

Comment:
this works, but for the future you can get the very same result using the kubectl format options, simplifying the command line. E.g.: `--no-headers=true -o custom-columns=...`

Reply:
I'll look into it